core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,17 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  /**
+   * A serialized form of <a
+   * href="https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUtils.java#L854C17">
+   * HIVE_COLUMN_STATS_OBJ</a>
+   */
+  public static final String HIVE_COLUMN_STATS_OBJ = "column-statistics-obj";
+
+  /**
+   * A serialized form of KLL sketch produced by the <a
+   * href="https://datasketches.apache.org/">Apache DataSketches</a> library
+   */
+  public static final String APACHE_DATASKETCHES_KLL_SKETCH = "apache_datasketches_kll_sketch";
```

Comment:
Add documentation to https://iceberg.apache.org/puffin-spec/#blob-types
and follow existing naming convention (hyphenated names).

Reply:
Sure. I will try to raise a iceberg-docs PR by tomorrow.

Reply:
Corresponding iceberg-docs PR : https://github.com/apache/iceberg-docs/pull/269